examples/demos/primary_keys_in_validation_results/spark/great_expectations/great_expectations.yml

```diff
@@ -94,9 +94,8 @@ data_docs_sites:
     site_index_builder:
       class_name: DefaultSiteIndexBuilder
 
-anonymous_usage_statistics:
+analytics_data:
```

Comment:
I think we might want to separate out the removal of the old platform and the introduction of the new one - might be too complex for a single PR?